technical-reports/resolver/terminology.md

```diff
@@ -1,103 +1,37 @@
-# Resolver Terminology
+# Terminology
 
-<dl>
-<dt>Resolver</dt>
-<dd>
+## <dfn>Resolver</dfn>
 
-A mechanism that combines token sets and applies modifiers to produce a final, resolved set of design tokens.
+The mechanism by which multiple possible values of design tokens are reduced to a single value, i.e. this module.
 
-</dd>
-<dt>Token Set</dt>
-<dd>
+## <dfn>Set</dfn>
 
-A collection of design tokens grouped together, often representing foundational tokens, component-specific tokens, or theme-specific tokens.
+Alternate design token values that are defined in [&lt;token-defs&gt;](#token-defs), intended to be merged in a specific order.
 
-</dd>
-<dt>Dimension</dt>
```

Comment:
“Dimension” was removed, because it is not only undefined, it also doesn’t appear in syntax in any way.

Open to keeping this around as a **non-normative definition,** but I think we should revisit how vital this concept is to a **specification** when it has no bearing on syntax or anything.